client/src/App.tsx

```diff
@@ -1,15 +1,19 @@
 import React, { useEffect, useState } from "react";
 import { Route, Switch } from "react-router-dom";
+import Login from "./pages/Login";
 import Home from "./pages/Home";
 import ScanStore from "./pages/ScanStore";
+import Receipt from "./pages/Receipt";
 import NotFound from "./pages/NotFound";
 
 function App() {
   return (
     <main>
       <Switch>
-        <Route path="/" component={Home} exact />
+        <Route path="/" component={Login} exact />
```

Comment:
If i try to visit /store endpoint directly will it first take me to Login page everytime?
IMO it should be -> if user is authenticated, load store and only if not authenticated, redirect to login!

Mostly we wont need to login the user again, just need credentials from gpay app IUUC @cszqwe ?!

Reply:
Yes, on successful login, user will be redirected from / -> /home. Currently I haven't checked user identity token since nothing is tied to users yet (now we are doing Cart so that requires auth to add/view). We can call getIdentity from any of our subpages not limited to / and microapps API will only pop out the dialog to grant us login permissions if the identity token issued has expired. (To test: We do intial login on /, on /home, clicking button "User Login" calls getIdentity again and grabs the already issued token instead of prompting user to login again.

Reply:
Will address in #64